lib/src/geojson.dart

```diff
@@ -251,7 +258,7 @@ abstract class Geometry extends GeoJSONObject {
 }
 
 abstract class GeometryType<T> extends Geometry {
-  T coordinates;
+  T? coordinates;
```

Comment:
Hm, so the problem here is that the way this is a union type with all the geometry types `T` we can't have an empty array be a valid value here?

It could be that we make a point to have a non-nullable `coordinates` value, but the other types can have empty arrays

Reply:
I think, I have a good way to handle this:
Let's make `coordinates` non-nullable, but with the security that if the value of `coordinates` is `null` when deserializing a GeoJSON file, it will be replaced by an empty array.